Prac_5/wimbledon.py

```diff
@@ -0,0 +1,46 @@
+"""
+Word Occurrences
+Estimate: 40 minutes
+Actual:   51 minutes
+"""
+
+
+FILE_NAME = "test.csv"
+
+
+def main():
+    file_data = get_data()
+    champion_countries, win_counts = scalp_data(file_data)
+    print_data(champion_countries, win_counts)
+
+
+def print_data(champion_countries, win_counts):
+    print("Wimbledon Champions:")
+    for champion in win_counts:
+        print(f"{champion} {win_counts[champion]}")
+    print(f"\nThese {len(champion_countries)} countries have won Wimbledon: ")
+    print(f", ".join(sorted(champion_countries)))
+
+
+def scalp_data(file_data):
+    win_counts = {}
```

Comment:
i dont think it a bad name but lindsey prefers for dict naming to be like win_to_counts I'm pretty sure, just so he doesnt mark you down